ui/bin/dockerui.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -eu
```

`set -e` -> If a command fails, stop, instead of trying to continue.
`set -u` -> Fail if there's a reference to a variable that isn't defined.

Reply:
added